Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -165,6 +172,13 @@ extension TestListViewController: EndpointDelegate {
                         self.testCycleList.testCycleList.insert(emptyCycle, at: 0)
                         self.testList.reloadData()
                     }
+                    self.testList.reloadData()
```

Comment:
Is this call needed? If not can we get rid of it. If it is needed can we get rid of the call in the if statement 2 lines up.